db/seeds.rb

```diff
@@ -5,3 +5,31 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+#  Console command to load ::  load "#{Rails.root}/db/seeds.rb"
+
+ factory = Factory.create(name: Faker::Company.name, city: Faker::GameOfThrones.city)
```

Comment:
This file is not in a readable format.
Do one thing, create new file inside a lib folder. And move your seed inside that file, and call it here.
While creating a lib file, chunk down your code into small methods, which will create instances of your models.